etl-data-pipeline/src/main/java/gov/cdc/etldatapipeline/changedata/model/dto/patient/AddAuthUser.java

```diff
@@ -0,0 +1,37 @@
+package gov.cdc.etldatapipeline.changedata.model.dto.patient;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.datatype.jsr310.deser.LocalDateTimeDeserializer;
+import gov.cdc.etldatapipeline.changedata.model.dto.PatientOP;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDateTime;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class AddAuthUser {
+    @JsonProperty("ADD_USER_ID")
+    private Long addUserId;
+    @JsonProperty("add_time")
+    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd'T'HH:mm:ss.SSS", timezone = "UTC")
```

Comment:
timezone = "UTC" @alog-enquizit Should this be UTC ? or should leave it to local

Reply:
@alog-enquizit Let us talk on this as a team - so we are clear if we should use timestamp with correct timezone based on JD's need